### PR TITLE
RocoMamas and Panarottis add spiders

### PR DESCRIPTION
**Pull request by arrival-spring**
Both are under Spur Corporation and have identical store finders

Awaiting NSI update for categories

```
 'atp/brand/RocoGo': 4,
 'atp/brand/RocoMamas': 104,
 'atp/brand_wikidata/Q123034964': 108,
 'atp/category/missing': 108,
 'atp/field/image/missing': 108,
 'atp/field/name/missing': 108,
 'atp/field/operator/missing': 108,
 'atp/field/operator_wikidata/missing': 108,
 'atp/field/phone/invalid': 2,
 'atp/field/postcode/missing': 9,
 'atp/field/state/missing': 19,
 'atp/field/twitter/missing': 108,
 'atp/item_scraped_host_count/rocomamas.com': 108,
 'atp/nsi/brand_missing': 108,
```

### CODE REVIEW

File: locations/spiders/rocomamas.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                body_data = (
+                    '{"url":"restaurants?brandKey=ROCO,ROCOGO&countryCode='
+                    + country
+                    + '&tradingStatus=Open&filterHidden=true&expand=channels","method":"GET"}'
+                )
+                yield JsonRequest(
+                    url=url,
+                    headers={"Content-Type": "application/json"},
+                    method="POST",
+                    body=body_data,
```
Comment (Cj-Malone) [inline locations/spiders/rocomamas.py:49]: Consider using data instead of body

Comment (arrival-spring) [reply locations/spiders/rocomamas.py:49]: Thanks, although it seems that it doesn't work if I use data here

**Comment by CloCkWeRX** [comment]
#10030

**Comment by arrival-spring** [comment]
@Cj-Malone I see what you mean now, thanks

**Pull request merged**